pkgs/by-name/ne/netop/package.nix

```
@@ -0,0 +1,27 @@
+{ lib, libpcap, rustPlatform, fetchFromGitHub }:
+
+rustPlatform.buildRustPackage rec {
+  pname = "netop";
+  version = "0.1.4";
+
+  src = fetchFromGitHub {
+    owner = "ZingerLittleBee";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "sha256-Rnp2VNAi8BNbKqkGFoYUb4C5db5BS1P1cqpWlroTmdQ=";
+  };
+
+  LIBPCAP_LIBDIR = lib.makeLibraryPath [ libpcap ];
+  LIBPCAP_VER = libpcap.version;
+
+  cargoSha256 = "5vbv4w17DdaTKuF3vQOfv74I8hp2Zpsp40ZlF08qWlc=";
+
+  meta = with lib; {
+    description = "Network Top -- Help you monitor network traffic with bpf";
+    homepage = "https://github.com/ZingerLittleBee/netop";
+    changelog = "https://github.com/ZingerLittleBee/netop/raw/v${version}/CHANGELOG.md";
+    platforms = platforms.linux;
+    license = [ licenses.mit ];
```

When only having one license this is the preferred way of writing it.
```suggestion
    license = licenses.mit;
```